examples/job-shop-scheduling/jobShopScheduler.py

```diff
@@ -0,0 +1,268 @@
+# Copyright 2019 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+from bisect import bisect_right
+
+import dwavebinarycsp
+
+
+def get_jss_bqm(job_dict, max_time=None):
+    """Returns a BQM to the Job Shop Scheduling problem.
+
+    Args:
+        job_dict: A dict. Contains the jobs we're interested in scheduling. (See Example below.)
+        max_time: An integer. The upper bound on the amount of time the schedule can take.
+
+    Returns:
+        A dimod.BinaryQuadraticModel. Note: The nodes in the BQM are labelled in the format,
+          <job_name>_<task_number>,<time>. (See Example below)
+
+    Example:
+        'jobs' dict describes the jobs we're interested in scheduling. Namely, the dict key is the
+         name of the job and the dict value is the ordered list of tasks that the job must do.
+
+        It follows the format:
+          {"job_name": [(machine_name, time_duration_on_machine), ..],
+           "another_job_name": [(some_machine, time_duration_on_machine), ..]}
+
+        >> # Create BQM
```

Comment:
Isn't the default Python prompt `>>>`?

Reply:
You're right, it is. I've used iPython for too long that I don't see this prompt anymore (that's the excuse I'm going with!)

Reply:
In that case I would recommend you do this:
```
>>> x = \
...     2
>>> 
>>> sys.ps1 = '>> '
>> sys.ps2 = '.. '
>> 
>> x = \
..     2
>> 
```

Reply:
Or, maybe even:
```
>>> sys.ps1 = "I'm to cool for triple-angle-brackets, I use >> "
I'm to cool for triple-angle-brackets, I use >> 
I'm to cool for triple-angle-brackets, I use >> # better ?
... 
I'm to cool for triple-angle-brackets, I use >> 
```
:smile: